### ISSUE TITLE
How to open the command console in Latin American Keyboard

### ISSUE THREAD

**Issue by SrLicht** [StunlockStudios/vrising-dedicated-server-instructions#37]
With which button on the Latin American keyboard (ARG) can I open the console? I have tried with all the buttons.

**Comment by MiguelMachado-dev** [comment]
`SHIFT + ´ + ' (which is ' or " under ESC)`

**Comment by xolan** [comment]
Yeah I had to remap a key to call `KC_TILD` on my nordic iso layout. Any combination of modifier keys would not work. Lucky to have a QMK-compatible keyboard...
